prime-angular-frontend/src/app/modules/adjudication/shared/components/health-authority-table/health-authority-table.component.ts

```diff
@@ -0,0 +1,56 @@
+import { AdjudicationRoutes } from '@adjudication/adjudication.routes';
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { MatTableDataSource } from '@angular/material/table';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Role } from '@auth/shared/enum/role.enum';
+import { Config } from '@config/config.model';
+import { ConfigService } from '@config/config.service';
+import { HealthAuthorityEnum } from '@shared/enums/health-authority.enum';
+
+@Component({
+  selector: 'app-health-authority-table',
+  templateUrl: './health-authority-table.component.html',
+  styleUrls: ['./health-authority-table.component.scss']
+})
+export class HealthAuthorityTableComponent implements OnInit {
+  public dataSource: MatTableDataSource<Config<number>>;
+
+  @Output() public route: EventEmitter<string | (string | number)[]>;
+
+  public columns: string[];
+
+  public Role = Role;
+  public AdjudicationRoutes = AdjudicationRoutes;
+  public haCode: HealthAuthorityEnum;
+
+  constructor(
+    protected configService: ConfigService,
```

Comment:
Access modifiers should be `private`.  You only need to use `protected` for DI if you were using inheritance and passing this through to the base class using `super(...)`

Reply:
I think i was looking at another class with DI and just saw them and thought hmm i guess we should be using protected instead of private again